test/gcp/packer_gcp_basic_example_test.go

```diff
@@ -23,7 +23,7 @@ var DefaultRetryablePackerErrors = map[string]string{
 var DefaultTimeBetweenPackerRetries = 15 * time.Second
 
 // Regions that don't support n1-standard-1 instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1"}
+var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
```

Comment:
It seems that there are many more regions that don't support n1-standard instances, for example: me-west1, europe-west9, all other us-east5 (not just -a) and many more : https://cloud.google.com/compute/docs/regions-zones

I was wondering if maybe this list should be done differently and include only the regions to test? What is the reason the n1 instance is used here? Can a different instance type be used?